free-1.md

```diff
@@ -10,34 +10,48 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This DSL would help users learn more about flavor pairings and make suggestions if given specific foods. The users would be able to figure out foods or beverages that would pair well with a given item. For example, if a person were hosting a dinner party and didn't know what to serve with the main dish, they could receive suggestions about flavors or dishes that would be good accompaniments. 
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Flavor pairing is a unique domain and a DSL would help to educate users about successful pairings. It is appropriate for users because they would be able to interact with pairings in various ways (learning about more pairings, getting pairing suggestions, etc).
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I am very interested in food science and flavor combinations. I talked with one of my friends who has similar interests and she said that she didn't always know what flavors would pair well together when trying to create a meal or dish. I had looked for similar pairing services in the past and was not satisfied with my findings.
+
```

Comment:
I like this idea because it seems more beneficial for the user when compared to using a recipe matching language like the one I described. The user actually builds upon his or her culinary knowledge.